src/quantum/azext_quantum/_validators.py

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+import os
+
+from .operations.workspace import WorkspaceInfo
+from .operations.target import TargetInfo
+
+def validate_workspace_info(cmd, namespace):
+    ### Makes sure all parameters for a workspace are available ##
```

Comment:
Minor, but making this a docstring instead of a comment would make it available to intelliSense.
```suggestion
    """
    Makes sure all parameters for a workspace are available.
    """
``` #Resolved